doc/locale/ja/LC_MESSAGES/install/cmake_options.po

```diff
@@ -0,0 +1,161 @@
+# -*- po -*-
+# Japanese translations for Groonga package.
+# Copyright (C) 2009-2022 Groonga Project
+# This file is distributed under the same license as the Groonga package.
+# Automatically generated, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Groonga 12.0.8\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2022-09-02 03:00+0900\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgid "CMake options"
+msgstr "CMake オプション"
+
+msgid "This section describes important CMake options."
+msgstr "このセクションはCMakeの重要なオプションについて記載しています。"
+
+msgid ""
+"To get more details about installing Groonga from the source by CMake on a "
```

Comment:
```suggestion
"To get more details about installing Groonga from the source with CMake on a "
```

I believe that CMake is a tool to build the Groonga with.
In English, when you want to explain the tool for the action, "with" is more common.

Ref:
https://www.myhappyenglish.com/free-english-lesson/2014/04/02/using-the-prepositions-by-with-english-grammar-lesson/